<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sfd::gui::main_window::MainWindow</class>
 <widget class="QMainWindow" name="sfd::gui::main_window::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Designer</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSaveTemplate">
     <property name="title">
      <string>Save Template</string>
     </property>
     <addaction name="actionSaveTemplate"/>
     <addaction name="actionManageTemplate"/>
    </widget>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuSaveTemplate"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionUpload"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionNewForm"/>
    <addaction name="separator"/>
    <addaction name="actionEditTranslTable"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="showSettingsDialog"/>
    <addaction name="actionDisplayPreview"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionProjectExplorer"/>
    <addaction name="actionProperties"/>
    <addaction name="actionToolbox"/>
    <addaction name="actionIssues"/>
    <addaction name="actionNonVisualComponents"/>
    <addaction name="separator"/>
    <addaction name="actionMenuToolbar"/>
    <addaction name="actionProjectToolbar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="showAboutDialog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="propertyEditorDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Property editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QDockWidget" name="documentsDock">
   <property name="windowTitle">
    <string>Documents</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <widget class="QDockWidget" name="pluginsDock">
   <property name="windowTitle">
    <string>Plugins</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6"/>
  </widget>
  <widget class="QDockWidget" name="undoStackDock">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Undo stack</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>Close Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionSaveTemplate">
   <property name="text">
    <string>Save Template</string>
   </property>
  </action>
  <action name="actionManageTemplate">
   <property name="text">
    <string>Manage Template...</string>
   </property>
   <property name="toolTip">
    <string>Manage Template</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionNewForm">
   <property name="text">
    <string>New Form</string>
   </property>
  </action>
  <action name="actionEditTranslTable">
   <property name="text">
    <string>Edit Translation Table</string>
   </property>
  </action>
  <action name="showSettingsDialog">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionDisplayPreview">
   <property name="text">
    <string>Display Preview</string>
   </property>
  </action>
  <action name="showAboutDialog">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionProjectExplorer">
   <property name="text">
    <string>Project Explorer</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionToolbox">
   <property name="text">
    <string>Toolbox</string>
   </property>
  </action>
  <action name="actionIssues">
   <property name="text">
    <string>Issues</string>
   </property>
  </action>
  <action name="actionNonVisualComponents">
   <property name="text">
    <string>Non Visual Components</string>
   </property>
   <property name="toolTip">
    <string>Non Visual Components</string>
   </property>
  </action>
  <action name="actionMenuToolbar">
   <property name="text">
    <string>Menu Toolbar</string>
   </property>
  </action>
  <action name="actionProjectToolbar">
   <property name="text">
    <string>Project Toolbar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>sfd::gui::main_window::MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
